src/lib.rs

```diff
@@ -141,6 +141,7 @@ pub const DEFAULT_OFFSET: u64 = 1;
 pub const DEFAULT_SEPARATOR: &str = ": ";
 pub const DEFAULT_TRACE_COLOR: Colour = Colour::Fixed(8); // grey
 pub const DEFAULT_WARN_COLOR: Colour = Colour::Fixed(11); // bright yellow
+pub const MODULE_PATH_UNKNOWN: &str = "unknown";
```

Comment:
isn't this just a thing if you have an older `log` in your dependency tree?
i remember having this, but got rid of it after upgrading `log` everywhere